images/macos/provision/core/commonutils.sh

```diff
@@ -6,6 +6,7 @@ source ~/utils/utils.sh
 
 # brew install
 binst_common_utils=(
+    aws-iam-authenticator
```

Comment:
Could you please split this PR to 2: Ubuntu and MacOS?
It will simplify review and merging

Reply:
Sounds good, I will do that.